### ISSUE TITLE
BUG: Subscribe button does not have sufficient contrast ratio

### ISSUE THREAD

**Issue by evelynfoy** [evelynfoy/the-cheese-and-wine-boutique#57]
Lighthouse is reporting that the Subscribe button does not have sufficient contrast ratio
- [ ] Button should be wine in keeping with site which will give a sufficient contrast rati
![Spike1Bug2Issue57](https://user-images.githubusercontent.com/35274182/184533508-72ab63d6-92e4-4d02-bef9-66247eeb75aa.jpg)
o
